src/services/UserService.ts

```diff
@@ -76,7 +77,12 @@ export class UserService {
   public async updateUser(editProfileRequest: EditProfileRequest, user: UserModel): Promise<UserModel> {
     return this.transactions.readWrite(async (transactionalEntityManager) => {
       const userRepository = Repositories.user(transactionalEntityManager);
-      return userRepository.updateUser(user, editProfileRequest.username, editProfileRequest.photoUrl,
+      let imageUrl = undefined;
+      if (editProfileRequest.photoUrl_base64) {
```

Comment:
also this naming scheme is a bit strange... why was it not `base64PhotoURL` or `photoURLBase64`

Reply:
maybe you can refactor this some other time